src/concepts/glean/accessing_glean_data.md

```diff
@@ -0,0 +1,71 @@
+# Accessing Glean data
+
+This document describes how to access Glean data from an SQL query, such as in [Redash](https://sql.telemetry.mozilla.org).
+
+## Selecting the correct table
+
+Each ping type is recorded in its own table, and these tables are named using `{application_id}.{ping_type}`.
+For example, for Fenix, the application id is `org_mozilla_fenix`, so its `metrics` pings are available in the table `org_mozilla_fenix.metrics`.
+
+## Selecting columns
+
+Columns of data are named using dot notation to organize them into groups.
```

Comment:
This is perhaps an oversimplification, since at the BigQuery level these are nested structs and the dot notation is used to descend into them.

How about "Fields are nested inside BigQuery `STRUCT`s to organize them into groups, and we can use dot notation to specify individual subfields in a query."

Reply:
That's exactly why I asked you to review :)  Thanks.